automation/build.sbt

```diff
@@ -43,37 +42,39 @@ Test / testForkedParallel := true
 Test / baseDirectory := (baseDirectory in ThisBuild).value
 
 /*
+ * Each test class is mapped to its own task.
  * Enables (true) or disables (false) parallel execution of tasks.
- * In not-forked mode: test classes are run in parallel in different threads, in same sbt jvm.
- * In forked mode: each test class runs tests in sequential order, in a separated jvm.
  */
-Test / parallelExecution := true
+Test / run / parallelExecution := true
 
 /**
   * disable sbt's log buffering
   */
 Test / logBuffered := false
 
 /**
-  * Control the number of forked JVMs allowed to run at the same time by
+  * Control the number of forked jvm allowed to run at the same time by
   *  setting the limit on Tags.ForkedTestGroup tag, which is 1 by default.
+  *
   *  Warning: can't set too high (set at 10 would crashes OS)
+  *  This is not number of threads in each jvm. That would be up to sbt.
   */
-Global / concurrentRestrictions := Seq(Tags.limit(Tags.ForkedTestGroup, 3))
+Global / concurrentRestrictions := Seq(Tags.limit(Tags.ForkedTestGroup, 4))
 
 /**
-  * Forked JVM options
+  * Forked jvm options
   */
-Test / javaOptions ++= Seq("-Xmx2G")
+Test / javaOptions ++= Seq("-Xmx4G")
+
+outputStrategy := Some(StdoutOutput)
 
 /**
- * copy system properties to forked JVM
+ * copy system properties to forked jvm
   */
-Test / javaOptions ++= Seq({
-  val props = System.getProperties
-  props.stringPropertyNames().asScala.toList.map { key => s"-D$key=${props.getProperty(key)}"}.mkString(" ")
-})
+Test / javaOptions ++= propertiesAsScalaMap(System.getProperties).map{ case (key,value) => "-D" + key + "=" +value }.toSeq
```

Comment:
why change this?  how did you test it?

Reply:
to test what they are, in sbt console, run `show test:javaOptions`